widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/SecureConversations.kt

```diff
@@ -90,3 +91,21 @@ internal fun SecureMessagingConfirmationScreenTheme(pallet: ColorPallet):
         )
     }
 }
+
+internal fun SecureMessagingTheme(pallet: ColorPallet): SecureMessagingTheme {
+    return pallet.let {
+        SecureMessagingTheme(
+            unavailableStatusBackground = LayerTheme(fill = it.systemNegativeColorTheme),
+            unavailableStatusText = BaseLightColorTextTheme(it),
+            bottomBannerBackground = LayerTheme(fill = it.baseNeutralColorTheme),
+            bottomBannerText = BaseNormalColorTextTheme(it),
+            bottomBannerDividerColor = it.baseShadeColorTheme,
+            topBannerBackground = LayerTheme(fill = it.baseNeutralColorTheme),
+            topBannerText = BaseDarkColorTextTheme(it),
+            topBannerDividerColor = it.baseShadeColorTheme,
+            topBannerDropDownIconColor = it.baseNormalColorTheme,
+            mediaTypeItems = DefaultMediaTypeItemsTheme(it)
+        )
```

Comment:
@DavDo please double check that those are correct colors

Reply:
Looks correct 👍

Reply:
Thank you!